docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,512 @@
 .. highlight:: psql
 .. _recover:
+.. _conf-env:
 
-====================================
-Troubleshooting with Crate-Node CLI
-====================================
+===================================
+Troubleshooting with Crate node CLI
+===================================
+This part of the CrateDB documentation is based on the `Elasticsearch
+Reference [7.2] <https://www.elastic.co/guide/en/elasticsearch/reference/current/node-tool.html>`_.
 
-The crate-node CLI enables you to perform certain unsafe operations
-on a node that are only possible while it is shut down. This command allows you
-to adjust the role of a node and may be able to recover some data after a
-disaster.
+The Crate node CLI is a powerful tool that allows you to change the cluster
+state of a node in order to repair a broken node or an entire broken cluster.
 
-If a CrateDB cluster unrecoverably looses the majority of the master eligible
-nodes, the crate-node CLI provides a way to utilize the state of the remaining
-master nodes to form a new cluster.
+With the CLI, your enabled to form a new CrateDB cluster, if a cluster
+irrecoverably loses the majority of the master-eligible nodes. To form a new
+cluster, the Crate node CLI allows you to use the state of the remaining master
+nodes.
 
-Nodes may be temporarily stopped for maintenance reasons or systems
-failures. Once the issues on the nodes are resolved, a restart is necessary
-to let them join the cluster again. Unfortunately this is not always
-possible. A node might be unrepairable and does not start-up,
-so if the cluster is still available the node can be simply replaced by
-a fresh node on a new host which will replace the broken node.
+This solution includes certain unsafe operations that are only possible while a
+node is shut down.
 
-crate-node CLI
-==============
+You can temporarily shut down nodes for several reasons, e.g., maintenance or
+systems failure. Once a node is shut down, you need to restart it to let the
+node rejoin the cluster. However, rejoining nodes isn’t always possible. For
+instance, a node could be irreparable and would not restart. With the cluster
+still available, you can replace a broken node by a new node on a new host.
+
+Modifying a node
+================
+
+. NOTE::
+
+  You can only perform the following operations after you have shut down the
+  affected node.
+
+The ``crate-node`` command enables you to adjust the role of a node and may also
+be able to recover some data after a disaster.
+
+You first have to call up the node, in order to modify it. Use these arguments:
```

Comment:
if my understanding of this section is accurate, then I think it needs rewriting a little to make it clear that (a) this is a command synopsis, and that the list afterward is a summary of what is possible (which you will then go on to explain in more detail in the following subsections)